test/parallel/test-tls-multi-key.js

```diff
@@ -2,52 +2,162 @@
 const common = require('../common');
 const assert = require('assert');
 
+// Test multi-identity ('key')/multi-algorithm scenarios.
+
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
 const tls = require('tls');
 const fs = require('fs');
 
-const options = {
+// Key and cert are ordered as ec, rsa.
```

Comment:
This test also has different tiers of cert chain. 
```
ec-cert(agent2):ECC (selfsigned)
agent1(agent1RSA):RSA -> ca1:RSA (selfsigned)
```
I think it is better to change the same tier level of the cert chain if we change this test.